docs: component updates week 39 DOC-2260 PAC-3230 (#8230)
spectrocloud/librarium: docs/docs-content/release-notes/release-notes.md

<ReleaseNotesVersions />

## September 26, 2025 - Component Updates {#component-updates-2025-39}

> The following components have been updated for Palette version 4.7.20 to 4.7.X.

Reviewer: ```suggestion
The following components have been updated in Palette versions 4.7.20 - 4.7.X.
```
Maybe? I'm trying to think of a way to make it clearer that the component updates are for a range...
